### PR TITLE
Add a ‘/‘ after prefix if it is not empty, and fix bug for empty stri…

### PR DESCRIPTION
**Pull request by NorwegianKiwi**
…ng. Add tests for prefix=null and prefix=“”.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ import functionArgNames from './functionArgNames';
 
 let actionNum = 0;
 const getActionType = (prefix, actionName) =>
-(prefix != null ? prefix : "") +
+(prefix ? prefix + "/" : "") +
```
Comment (espen42) [inline src/index.js]: Yes, might as well standardize the prefix along the duck convention. But maybe it'd be nice with a check for whether there is already a slash at the end? I'll pull this and make that. :)
